clientv3/retry.go

```diff
@@ -66,15 +66,17 @@ func (c *Client) newRetryWrapper(isStop retryStopErrFunc) retryRpcFunc {
 			if logger.V(4) {
 				logger.Infof("clientv3/retry: error %v on pinned endpoint %s", err, pinned)
 			}
+			// do not switch endpoint when server is stopped
+			// (should exit on non-transient error)
+			if isStop(err) {
```

Comment:
why this is non-transient error?

Reply:
@xiang90 For instance, `rpctypes.ErrEmptyKey` should not be retried.
There's a test for this https://github.com/coreos/etcd/blob/master/clientv3/integration/kv_test.go#L36-L52.